### DISCUSSION TITLE
Separate List from Stars

### DISCUSSION THREAD

**Discussion by meetxpress** [community/community#11041] [Other Feature Feedback, Questions, & Ideas]
The list thing is working so nice, and I love it. 
There is a suggestion that it would be great if the repositories we add to the list should not be visible in the starred repositories.

**Comment by haxpor** [comment]
I second this. Thanks for given this feedback!

Additionally, personally I don't expect Lists to be landed in the same place as Stars. Listed repositories for me is for bookmarking, and grouping together. Somewhat more like a former self of Stars in which we star repository (sometimes) in the past because we have no choice to bookmark them for later. Now Lists are here to solve. It would be good to separate them.
